zos_version.go

```diff
@@ -34,3 +34,22 @@ func (s *Substrate) GetZosVersion() (string, error) {
 
 	return zosVersion, nil
 }
+
+// CreateTwin creates a twin
+func (s *Substrate) SetZosVersion(identity Identity, version string) (string, error) {
+	cl, meta, err := s.getClient()
+	if err != nil {
+		return "", err
+	}
+
+	c, err := types.NewCall(meta, "TfgridModule.set_zos_version", version)
```

Comment:
Wrap is call in a council proposal, since we will be initiating this call by a key that will be part of the council. It will require other council members to approve this call.

`Council.Propose(threshold, actual_call, lengthBound)`